lib/main/atom/commands/initializeConfig.ts

```diff
@@ -0,0 +1,66 @@
+import {commands} from "./registry"
+import {resolveModule} from "../../../client/clientResolver"
+import {execFile} from "child_process"
+
+commands.set("typescript:initialize-config", () => {
+  return async e => {
+    const projectDirs = atom.project.getDirectories()
+
+    if (projectDirs.length === 0) {
+      e.abortKeyBinding()
+      return
+    }
+
+    let editor
+    let currentPath
+
+    try {
+      editor = atom.workspace.getActiveTextEditor()
+
+      if (!editor) {
+        throw new Error("There is no active text editor available.")
+      }
+
+      currentPath = editor.getPath()
+
+      if (!currentPath) {
+        throw new Error("There is no active filepath available.")
+      }
+    } catch (e) {
+      console.error(e.message)
```

Comment:
:art: TBH, if all it does is `console.error`, then `try`/`catch` seems a bit pointless, it's more lines of code for the same effect as
```ts
if (!a) {
  console.error("message A")
  return
}
// ...
if (!b) {
  console.error("message B")
  return
}
```

If it would do some error handling besides logging to console, it would make much more sense. Which it can, by the way, doing `e.abortKeyBinding()` as appropriate, or showing a nice user-friendly message that something went wrong (via `atom.notifications.addWarning` for example)

Note that you can differentiate different errors by their message, f.ex.
```ts
try {
    // ...
    throw new Error("ENOPROJECT")
    // ...
    throw new Error("ENOEDITOR")
  } catch (e) {
    switch (e.message) {
      case "ENOPROJECT":
      case "ENOEDITOR":
        e.abortKeyBinding()
        return // we handled those, yay
      default:
        // required because you can never be sure what is actually thrown
        if (e.stack) {
          // because we want to be sure it has stack,
          // otherwise weirdness will ensue
          atom.notifications.addFatalError("Something went wrong", {
            detail: e.message,
            dismissable: true,
            stack: e.stack,
          })
        } else {
          atom.notifications.addError("Something went wrong and we don't know where", {
            detail: e.toString(),
            dismissable: true,
          })
        }
        return
    }
  }
```
(note: this is very general, here you can safely assume that `e` is instance of `Error` and has `stack` defined)